apps/explorer/src/app/components/vote-icon/vote-icon.tsx

```diff
@@ -18,14 +20,27 @@ export interface VoteIconProps {
  */
 export function VoteIcon({
   vote,
+  useVoteColour = true,
   yesText = 'For',
   noText = 'Against',
 }: VoteIconProps) {
   const label = vote ? yesText : noText;
-  const bg = vote ? 'bg-vega-green-550' : 'bg-vega-pink-550';
   const icon: IconName = vote ? 'tick-circle' : 'delete';
-  const fill = vote ? 'vega-green-300' : 'vega-pink-300';
-  const text = vote ? 'vega-green-200' : 'vega-pink-200';
+  const bg = useVoteColour
```

Comment:
nitpick (non-blocking): not the biggest fan of nested ternaries, I'd use a map with 3 properties.

Reply:
It's a fair criticism, fixed